src/components/play-bar/desktop/PlayBar.js

```diff
@@ -321,6 +320,9 @@ class PlayBar extends Component {
       playButtonStatus = 'play'
     }
 
+    const favoriteAndRepostIsDisabled = !uid || isOwner
+    const favoriteText = favorited ? 'Unfavorite' : 'Favorite'
```

Comment:
In preparation for i18n, we are trying to put all user-facing text in a messages object, typically defined above the component, so something like
```typescript
const messages = {
  favorite: 'favorite',
  unfavorite: 'unfavorite'
}
```
and then used in the component like: 
```
const favoriteText = favorited ? messages.unfavorite : messages.favorite
```

Reply:
gotcha, thanks! i think this pattern is fine until we figure out what we're going to use. sadly i do not have any suggestions there (the last i18n experience i had was at Airbnb where they had built an intense in house solution)